roles/controller_casc_from_aap/tasks/all.yml

```diff
@@ -37,7 +37,7 @@
       loop_control:
         loop_var: asset_item
         label: asset_item
-  when: "['all', 'credentials', 'credential_types', 'execution_environments', 'inventory_sources', 'notification_templates', 'projects', 'teams', 'users'] | intersect(input_tag) | length > 0"
+  when: "['all', 'credentials', 'credential_types', 'execution_environments', 'inventory_sources', 'notification_templates', 'teams', 'users','roles'] | intersect(input_tag) | length > 0"
```

Comment:
"roles" is not necessary because the roles playbook is called from users and teams.